### ISSUE TITLE
Overriding views does not work (as expected)

### ISSUE THREAD

**Issue by texnixe** [getkirby/kirby#2346]
**Describe the bug**  
According to the docs, it should be possible to override default Panel views: https://getkirby.com/docs/reference/plugins/extensions/panel-views#override-default-views. However, this does not work, at least not as described. I also tried several other options, but couldn't get it to work.

```
import UsersView from "./components/UsersView.vue";
panel.plugin('texnixe/users-view', {
  components: {
    "k-users-view": UsersView
  }
});
```
This is the code I used, with UsersView.vue being more or less a copy of the original view, only layout changed. I can easily add a copy of the view by registering it as a new view, but that defeats the purpose.

**Expected behavior**  
Overriding a default view should be possible or the docs should be updated if the way to achieve this is different.

**Kirby Version**  
Tested with 3.3.1 and 3.3.2-rc1

@bastianallgeier tested this as well a few days ago without success.

**Comment by afbora** [comment]
Reproduced too. I was try to extend `k-settings-view` component, i couldn't. Then i gave up, thinking I was wrong.

**Comment by bastianallgeier** [comment]
✅

**Comment by bastianallgeier** [comment]
I had to revert it :( There are a few more issues with this.

**Comment by distantnative** [comment]
I am afraid this isn't a bug, just wrong docs...

What seems to work for me:

```javascript
panel.plugin("my/view-plugin", {
  views: {
    users: {
      link: "/users",
      icon: "heart",
      menu: true,
      component: {
        template: `
          <k-view class="k-todos-view">
            <k-header>
                Todos
            </k-header>

            <!-- your app goes here -->

          </k-view>
        `
      }
    }
  }
});
```

Can anyone confirm?

**Comment by afbora** [comment]
Works for me too 👍

**Comment by distantnative** [comment]
🙌

**Comment by iskrisis** [comment]
I think its two connected issues. It's same as this one https://github.com/getkirby/kirby/issues/2506

One @distantnative mentions has to do with "views" component helpers. Specifically everything loaded using 
```
panel.plugin("my/view-plugin", {
  views: {
    users: {
    }
  }
});
```
Here it rewrites component name and route https://github.com/getkirby/kirby/blob/7525553b8d9976d6ed08b702f62fc3d368116777/panel/src/config/plugins.js#L87 so it's always ```k-users-plugin-view``` and link/route is ```/plugin/users``` so it can never overwrite the default ```k-users-view``` and users view is reachable at ```/plugins/users```.

----

But i think @texnixe was right that overwriting the component directly
```
panel.plugin('texnixe/users-view', {
  components: {
    "k-users-view": UsersView
  }
});
```
Should work. 

I think i've tracked it down and its seems its wierd import order issue. I had no idea but ES6 import [seem to not be guaranteed](https://stackoverflow.com/questions/35551366/what-is-the-defined-execution-order-of-es6-imports). So this buch of imports
https://github.com/getkirby/kirby/blob/7525553b8d9976d6ed08b702f62fc3d368116777/panel/src/main.js#L14-L25
Is not odered as we expect. 
![Screenshot 2020-10-02 at 08 11 16](https://user-images.githubusercontent.com/4954323/94893520-ac22f400-0487-11eb-9d94-fa095716e914.png)
Basically routes.js is imported/run sooner than plugins.js. That means that all core routes defined in router.js are not gonna register components from plugins and will use the default core components no matter what.

This is not a smart fix but if you import something from ```plugins.js``` to ```router.js```
```
plugins.js
export default components;
```
and in top of ```router.js```
```
router.js
import components from "./plugins.js";
```
Webpack changes the ordering
![Screenshot 2020-10-02 at 08 16 53](https://user-images.githubusercontent.com/4954323/94893590-db396580-0487-11eb-915f-abaf0ce9ba85.png)
 to what we expect and overwriting of default starts to work as is documented here https://getkirby.com/docs/reference/plugins/extensions/panel-views#override-default-views
